doc/examples/color_exposure/plot_ihc_color_separation.py

```diff
@@ -61,9 +61,9 @@
 zdh = np.dstack((np.zeros_like(h), d, h))
 
 fig = plt.figure()
-ax = plt.subplot(1, 1, 1, sharex=ax0, sharey=ax0, adjustable='box-forced')
-ax.imshow(zdh)
-ax.set_title("Stain separated image (rescaled)")
-ax.axis('off')
+axis = plt.subplot(1, 1, 1, sharex=ax[0], sharey=ax[0], adjustable='box-forced')
```

Comment:
@ahojnnes I meant that `1, 1, 1, sharex=ax[0], sharey=ax[0]` is redundant here. `fig, axes` might be initialized in a single line (same as in other examples).